sematic/ui/packages/common/src/pages/RunDetails/FunctionSection.tsx

```diff
@@ -0,0 +1,99 @@
+import styled from '@emotion/styled';
+import Box from '@mui/material/Box';
+import { DateTimeLong, Duration } from 'src/component/DateTime';
+import Docstring from 'src/component/Docstring';
+import Headline from 'src/component/Headline';
+import ImportPath from 'src/component/ImportPath';
+import MoreVertButton from 'src/component/MoreVertButton';
+import PipelineTitle from 'src/component/PipelineTitle';
+import RunReferenceLink from "src/component/RunReferenceLink";
+import { SuccessStateChip } from 'src/component/RunStateChips';
+import Section from 'src/component/Section';
+import TagsList from 'src/component/TagsList';
+import theme from 'src/theme/new';
+
+const StyledSection = styled(Section)`
+    height: fit-content;
+    min-height: 200px;
+    position: relative;
+
+    &:after {
+        content: '';
+        position: absolute;
+        bottom: 0;
+        height: 1px;
+        width: calc(100% + ${theme.spacing(10)});
+        margin-left: -${theme.spacing(5)};
+        background: ${theme.palette.p3border.main};
+    }
+    
+`;
+
+const StyledVertButton = styled(MoreVertButton)`
+    position: absolute;
+    right: 0;
+    top: 12px;
+    transform: translate(50%,0);
+`;
+
+const BoxContainer = styled(Box)`
+    display: flex;
+    flex-direction: row;
+    height: 35px;
+    align-items: center;
+    margin-bottom: ${theme.spacing(3)};
+
+    & .Info {
+        margin-left: ${theme.spacing(2)};
+
+        > div {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            column-gap: ${theme.spacing(2)};
+        }
+    }
+`;
+
+const RunStateContainer = styled(Box)`
+    flex-grow: 0;
+    width: 25px;
+`
+
+const ImportPathContainer = styled(Box)`
+    margin-bottom: ${theme.spacing(3)};
+`
+
+const StyledRunReferenceLink = styled(RunReferenceLink)`
+    font-size: ${theme.typography.fontSize}px ;
+`;
+
+const FunctionName = PipelineTitle
+
+const FunctionSection = () => {
+    return <StyledSection>
+        <Headline>Function Run</Headline>
```

Comment:
Does the headline need to be in all caps?

Reply:
Similar to other section titles, they are all capitalized to the first letter using CSS `text-transform: capitalize`. So even if we write the text in lower cases, they will still be rendered in upper cases that way. But this behavior can be overridden if so desired. 

Personally, I feel having them capitalized is fine. Why do you think all caps should not be preferred?